lib/tools/common/patching_parallel.py

```
@@ -0,0 +1,1244 @@
+            try:
+                result = subprocess.run(
+                    ["umount", self.mount_path],
+                    capture_output=True, text=True, check=False
+                )
+                if result.returncode != 0:
+                    log.warning(f"Failed to unmount {self.mount_path}: {result.stderr}")
+                    success = False
+                else:
+                    log.debug(f"Unmounted {self.mount_path}")
+                    self.is_mounted = False
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)